source/blog/2018-11-29-december-event

```diff
@@ -0,0 +1,33 @@
+---
+title: First Annual DecEmber Event!
+author: Melanie Sumner
+tags: Recent Posts, 2018
+responsive: true
+---
+
+December is a great month to support your friendly Ember.js learning team! 
+
+Maybe December is a busy month for you, because of the holidays and family obligations- and that’s totally okay- we hope you have a great time! But for some, December is a time of the year with a lot of downtime. Maybe that’s you! Are you wondering what to do with all your vacation time in December? We’ve got ya covered! 
+
+This year, the Ember Learning Core Team is sponsoring our first annual DecEmber, a hackathon event specifically targeted at Learning Team efforts. For an extra bit of awesome, it’s running at the same time as 24 Pull Requests, so contributions to DecEmber will count for both!
+
+## Participation Rules
+To qualify for any of our random assortment of thank you goodies (that we’ve delightfully dubbed Embercouragement), you must complete at least 5(five) participation activities for repositories in the ember-learn org between December 1 and December 31, 2018. The more contributions, the bigger the thank you! 
```

Comment:
`The more contributions, the bigger the thank you!` this sentence is missing from the `.md` file. Do you want to include it? If so we will need to change it to:

```
## Participation Rules

<!--alex ignore bigger-->
To qualify for any of our random assortment of thank you goodies (that we’ve delightfully dubbed Embercouragement), you must complete at least 5(five) participation activities for repositories in the ember-learn org between December 1 and December 31, 2018. The more contributions, the bigger the thank you! 
```